app/templates/base-en.html

```diff
@@ -58,32 +58,22 @@
     {%- set language = { 'languages':
         [
             {
-                'url': '/en' + page_url,
-                'ISOCode': 'en',
-                'text': 'English',
-                'buttonAriaLabel': 'Language selector. Current language: English',
```

Comment:
We need to be careful about removing stuff like this, AriaLabels are used for screen reader software to keep the page accessible and without them the page might not be properly accessible. I won't block on this now since if we do need these labels then the wording probably needs designing and updating, but I'll add a ticket to re-check the whole page.